tests/release_tests/test_launch/README.md

```diff
@@ -0,0 +1,7 @@
+# Launch Release Tests
+
+These tests are intended to be run before any release of Launch, to ensure common use cases haven't regressed. The tests shouldn't mock anything related to the wandb server, and should make actual API calls the same way a user would.
+
+To run these tests, you need to be logged into a Weights & Biases account that has access to the entity `launch-release-testing`. You will also need to have AWS credentials - the env vars `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` should be set.
```

Comment:
Can we also make the base url an option to pass in, I'd like to test against saas, and a local instance that we always have available